server/test/login.test.js

```diff
@@ -0,0 +1,45 @@
+import app from '../server.js';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+const expect = chai.expect;
+const should = chai.should();
+
+chai.use(chaiHttp);
+
+describe('POST /api/auth/sigin', () => {
+    it("Should login user, return auth token, and status code of 200", (done) => {
+        chai.request(app)
+            .post('/api/auth/signin')
+            .send({
+                "email": "test@email.com",
+                "password": "testpassword"
+            })
+            .end((err, res) => {
+                if (err)
+                    return done(err);
+                res.should.have.status(200);
+                expect(res.body.error).to.be.equal(undefined);
+                expect(res.body.token).to.not.be.null;
+                console.log('Res body message: ', res.body.message);
+                console.log('Token: ', res.body.token);
+                done();
+            })
+    })
+
+    it("Should not login user and return a status code of 500", (done) => {
```

Comment:
I would branch this out in two cases. "Should not login when user enter an email that does not exist in the system" and "Should not login when password is not correct"

Reply:
Added the requested changes